crates/opencascade-sys/build.rs

```diff
@@ -35,7 +35,7 @@ fn main() {
 
     cxx_build::bridge("src/lib.rs")
         .cpp(true)
-        .flag_if_supported("-std=c++11")
+        .flag_if_supported("-std=c++14")
```

Comment:
I'd prefer if we don't change this just for the sake of being able to use `make_unique`.

Reply:
Is there any reason to use c++11? Its still a 11 year old standard at this point.

Reply:
My reasoning is just that if it works with c++11 already, there's no reason to raise the requirements arbitrarily. I don't imagine we'll be writing a lot of exotic C++ code, it's mostly just wrappers for constructors.

C++11 has been around long enough that it's well supported by all the major compilers out there. C++14 is probably around that point as well, but I'd rather not move to it unless there's a very compelling reason to do so.

Reply:
Ill push an update that works for C++11. Thanks for explaining the reasoning.

Reply:
Sorry for the churn on that.